composer/modules/web/source-gen/SourceGenTemplate.java

```diff
@@ -969,6 +975,13 @@ private void modifyNode(JsonObject node, String parentKind) {
                         .get(0).getAsJsonObject().get("text").getAsString());
             }
         }
+
+        if (kind.equals("Deprecated") &&
```

Comment:
Always compare constant string part with the variable to avoid NPEs. `if("Deprecated".equals(kind))`.